### DISCUSSION TITLE
Can't sign up since my school doesn't give student email addresses to students

### DISCUSSION THREAD

**Discussion by fkeyzuwu** [community/community#76874] [GitHub Education]
### Hi everyone,

Hey, I study in Rimon School of Music. However, in the registration process for applying for GitHub Education, you do not allow for emails that aren't school emails. But my school does not give out email addresses to students, only to teachers. Is there avoid having to give a school email?

**Comment by hackelite01** [comment]
Hello, @fkeyzuwu, you can use your regular email address for the GitHub Education registration. To verify your affiliation with Rimon School of Music, you can utilize your school ID during the GitHub Education application process. There is no need for a school-issued email; your school ID will serve as the verification.

**Comment by notsointresting** [comment]
Yes, you can still apply for GitHub Education even if you don't have a school email. GitHub allows you to provide other forms of academic status proof. Here's how you can do it:

1. Visit the GitHub Student Developer Pack page.
2. Click on "Get your Pack".
3. If you're not logged into GitHub, you'll be asked to create an account or log in.
4. Once you're logged in, you'll be taken to a form where you can provide your academic proof.
5. Upload your proof of enrollment (this could be a screenshot of your school's portal, your ID card, or any other document that shows you're currently a student).
6. Submit the form and wait for GitHub to review your application.

Remember, the proof must show that you're currently enrolled in an academic program.
